modules/apps/notification/notification-web/src/main/resources/META-INF/resources/js/components/SettingsContainer/UserNotificationSettings.tsx

```diff
@@ -97,9 +108,47 @@ export function UserNotificationSettings({
 		setRolesList([]);
 	};
 
+	const getUserGroups = async () => {
```

Comment:
Can we move this logic into a new userGroupsUtil.ts file?  Similar to what rolesUtil.ts does?

I will do this, since I've already created the file for my PR.